ZeroWasteFriendly/Helpers/Constants.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Constants.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 20/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import CoreLocation
+
+enum Constants {
+
+    enum EcoMap {
+        static let katowiceCoordinates = UserLocation(location: CLLocation(latitude: 50.257564, longitude: 19.021145))
+        static let kilometersRadius: CLLocationDistance = 1000
```

Comment:
I actually think that these constants will be used only once in a single file. In that kind of cases I'd prefer to have private extensions to UserLocation and CLLocationDistance there